app/bundles/EmailBundle/Controller/BatchEmailController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
```

Comment:
Please add `declare(strict_types=1);` to all new PHP classes.
```suggestion

declare(strict_types=1);

```